plugins/Marketplace/lang/en.json

```diff
@@ -5,78 +5,78 @@
+        "InstallingPlugin": "Installing %s…",
         "InstallPurchasedPlugins": "Install purchased plugins",
         "LastCommitTime": "(last commit %s)",
         "LastUpdated": "Last Updated",
         "License": "License",
         "LicenseKey": "License key",
-        "ManageLicenseKeyIntro": "You can extend Matomo's capabilities using our premium plugins. To install premium plugins, you need a valid license key, which can be obtained in two ways: you can start a trial for your selected plugin in the %1$smarketplace%2$s, or purchase a plugin subscription on %3$sour website%4$s. Starting a trial in the marketplace will automatically create a license key and add it to this instance. If you purchase a subscription on the website, you will receive a license key that you can add it in the field below. After your license key is added, you can update or remove it if needed.",
-        "LicenseKeyActivatedSuccess": "License key successfully activated!",
-        "LicenseKeyDeletedSuccess": "License key successfully deleted.",
+        "ManageLicenseKeyIntro": "Try a premium plugin from %1$sthe marketplace%2$s, or get a license on %3$sthe website%4$s first and add it below.",
+        "LicenseKeyActivatedSuccess": "License key activated.",
+        "LicenseKeyDeletedSuccess": "License key deleted.",
         "Exceeded": "Exceeded",
         "LicenseMissing": "License missing",
-        "LicenseMissingDeactivatedDescription": "The following plugins have been deactivated because you are using them without a license: %1$s. %2$sTo resolve this issue either update your license key, %3$sget a subscription now%4$s or deactivate the plugin.",
-        "PluginLicenseMissingDescription": "You are not allowed to download this plugin because there is no license for this plugin. To resolve this issue either update your license key, get a subscription or uninstall the plugin.",
+        "LicenseMissingDeactivatedDescription": "%2$sRenew your license to keep using %1$s, or %3$sget a new license%4$s.",
+        "PluginLicenseMissingDescription": "Renew your license or get a new one to download this plugin.",
         "LicenseExceeded": "License exceeded",
-        "LicenseExceededDescription": "The licenses for the following plugins are no longer valid as the number of authorized users for the license is exceeded: %1$s. %2$sYou will not be able to download updates for these plugins. To resolve this issue either delete some users or %3$supgrade the subscription now%4$s.",
-        "PluginLicenseExceededDescription": "You are not allowed to download this plugin. The license for this plugin is no longer valid as the number of authorized users for the license is exceeded. To resolve this issue either delete some users or upgrade your subscription now.",
+        "LicenseExceededDescription": "%3$sUpgrade your license%4$s or remove users to keep using %1$s. %2$sThen you can also get new versions.",
```

Comment:
I don't understand what the `%2$s` is doing here.

Reply:
Also adds a `<br />`

Reply:
Removing the whitespace then. :)